tests/specifications/googlefonts_test.py

```diff
@@ -853,13 +851,13 @@ def test_check_085():
   some_bad_values = ["APACHE", "Apache", "Ufl", "Ofl", "Open Font License"]
 
   for good in good_licenses:
-    print ("Test PASS: ".format(good))
+    print("Test PASS: ".format(good))
```

Comment:
I just realized the original code lacks `{}` :grin:

Reply:
Oops! So pyupgrade worked here and helped find a bug :)

I've added the `{}` and re-pyupgraded!